package.json

```diff
@@ -36,7 +36,7 @@
         "build": "nx run-many --target=build --all",
         "format": "pnpm recursive run format",
         "lint": "nx run-many --target=lint --all",
-        "lint:fix": "nx run-many --target=lint:fix --all",
+        "lint:fix": "nx run-many --target=lint --fix --all",
```

Comment:
Please consider removing this change again, and make sure that all modules work as expected, to keep running something within a module consistent with running it from root.

Reply:
Done in https://github.com/SAP/open-ux-tools/pull/813/commits/394d782198507cc923adec228708c1cc05244333